src/index.js

```diff
@@ -83,3 +84,29 @@ export function getUrl(portalUrl, skylink, options = {}) {
 
   return parsed.toString();
 }
+
+const SKYLINK_MATCHER = "(?<skylink>[a-zA-Z0-9_-]{46})";
+const SKYLINK_DIRECT_REGEX = new RegExp(`^${SKYLINK_MATCHER}$`);
+const SKYLINK_SIA_PREFIXED_REGEX = new RegExp(`^sia://${SKYLINK_MATCHER}$`);
+const SKYLINK_PATHNAME_REGEX = new RegExp(`^/${SKYLINK_MATCHER}`);
```

Comment:
Would it work to combine them into `(sia:\/\/|\/|)(?<skylink>[a-zA-Z0-9_-]{46})` ? 

https://regex101.com/r/YSBEXr/1